openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -38,7 +38,7 @@
     PRIVATE_VISIBILITY,
     VISIBILITY_PREFIX
 )
-from .image_helpers import get_profile_image_urls_for_user
```

Comment:
The `serializers` module gets imported  a lot from the `api` module already so it's leading to a circular import.  I think you should extract the function to `utils.py` and then call it from both the API and from the serializer  to resolve the issue.

Reply:
Sure, thanks, updated code

Reply:
There is still a problem with circular imports when placing the code in `utils.py`, do you think the proposed implementation using `AccountLegacyProfileSerializer` in `api.py` is appropriate?